demucs/stft_process.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import torch
+
+# To export your own STFT process ONNX model, set the following values.
+
+DYNAMIC_AXES = True                                 # Default dynamic axes is input audio (signal) length.
+NFFT = 1024                                         # Number of FFT components for the STFT process
+HOP_LENGTH = 256                                    # Number of samples between successive frames in the STFT
+INPUT_AUDIO_LENGTH = 16000                          # Set for static axes. Length of the audio input signal in samples.
+MAX_SIGNAL_LENGTH = 2048                            # Maximum number of frames for the audio length after STFT processed. Set a appropriate larger value for long audio input, such as 4096.
+WINDOW_TYPE = 'hann'                                # Type of window function used in the STFT
+PAD_MODE = 'reflect'                                # Select reflect or constant
+NORMALIZED = True                                   # Whether to normalize the window function
+STFT_TYPE = "stft_B"                                # stft_A: output real_part only;  stft_B: outputs real_part & imag_part
+ISTFT_TYPE = "istft_B"                              # istft_A: Inputs = [magnitude, phase];  istft_B: Inputs = [magnitude, real_part, imag_part], The dtype of imag_part is float format.
+export_path_stft = f"{STFT_TYPE}.onnx"              # The exported stft onnx model save path.
+export_path_istft = f"{ISTFT_TYPE}.onnx"            # The exported istft onnx model save path.
```

Comment:
What is used for? If this is for the generated model, I would suggest you use command argument, or at least environment variable, to allow customising this. If this makes no sense to be able to customize it, please remove the string interpolation.